src/Utils/LocalizeVector.js

```diff
@@ -1,5 +1,4 @@
-const from = undefined,
-      skipUpdate = true;
+var skipUpdate = true;
```

Comment:
I forgot babel is not here... so s/const/var/
And `from = undefined` gives a warning while compiling
`line 1  col 5  It's not necessary to initialize 'from' to 'undefined'.`